frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/create-local-volume-set-step/body.tsx

```diff
@@ -51,24 +51,29 @@ export const LocalVolumeSetBody: React.FC<LocalVolumeSetBodyProps> = ({
   storageClassName,
   nodes,
   allNodes,
+  defaultVolumeMode,
 }) => {
   const { t } = useTranslation();
   const [radio, setRadio] = React.useState(FilterDiskBy.ALL_NODES);
   const [activeMinDiskSize, setMinActiveState] = React.useState(false);
   const [activeMaxDiskSize, setMaxActiveState] = React.useState(false);
 
-  React.useEffect(() => {
-    // Update the nodes with allNodes when the component is rendered
-    dispatch({ type: 'wizard/nodes', payload: allNodes });
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
-
   const formHandler = React.useCallback(
     (field: keyof LocalVolumeSet, value: LocalVolumeSet[keyof LocalVolumeSet]) =>
       dispatch({ type: 'wizard/setCreateLocalVolumeSet', payload: { field, value } }),
     [dispatch],
   );
 
+  React.useEffect(() => {
+    // Update the nodes with allNodes when the component is rendered
+    dispatch({ type: 'wizard/setNodes', payload: allNodes });
+
+    // Update the pre-selected volume mode
+    formHandler('diskMode', defaultVolumeMode);
```

Comment:
For mcg default volume mode - filesystem,
For full deployment, volume mode - block